docs/controllers.md

```diff
@@ -48,3 +48,12 @@ Integral action, as well as gain scheduling based on turbine saturation, has bee
 simple proportional control appears sufficient currently. However, these may be enabled at a 
 later date if needed. The `proportional_gain` for the controller may be provided on instantiation,
 and defaults to `proportional_gain = 1`.
+
+### HybridSupervisoryControllerBaseline
```

Comment:
@genevievestarke could you check the documentation below and see if I've accurately portrayed the `HybridSupervisoryControllerBaseline`?

Reply:
@genevievestarke , just bumping this, are you OK with the description I've written for the `HybridSupervisoryControllerBaseline`? Feel free to update the description if you'd like.